deicode/scripts/_rpca.py

```diff
@@ -38,10 +38,12 @@ def sample_filter(val, id_, md): return sum(val) > min_sample_depth
     feature_loading = pd.DataFrame(opt.feature_weights, index=table.columns)
     feature_loading = feature_loading.rename(columns=rename_cols)
     feature_loading.sort_values('PC1', inplace=True, ascending=True)
+    feature_loading -= feature_loading.mean(axis=0)
 
     # Sample Loadings
     sample_loading = pd.DataFrame(opt.sample_weights, index=table.index)
     sample_loading = sample_loading.rename(columns=rename_cols)
+    sample_loading -= sample_loading.mean(axis=0)
```

Comment:
This doesn't have to be resolved now, but for the lucky future developer - it looks like there is some redundancy here compared to _method.py.  May want to redefine some functions to avoid copy-pasta fixes like this.

Reply:
definitely. need to merge between them.

Reply:
Another note for a future developer, there will eventually need to have unittests to test the contents of feature loadings and sample loadings -- this will become more important for regression testing to catch future bugs.

Reply:
These notes/suggestion will be lost - what about creating specific issues in the repo and adding a full description of what needs to be done + pointing here?